tests/clipboard/clipboard_plugin.test.ts

```diff
@@ -556,7 +561,10 @@ describe("clipboard", () => {
       setCellContent(model, "A2", "3");
       copy(model, "A1:B2");
       const htmlContent = model.getters.getClipboardContent()[ClipboardMIMEType.Html]!;
-      const expectedHtmlContent = `<div data-clipboard-id="${model.getters.getClipboardId()}"><table border="1" style="border-collapse:collapse"><tr><td style="">1</td><td style="">2</td></tr><tr><td style="">3</td><td style=""></td></tr></table></div>`;
+      const cbPlugin = getPlugin(model, ClipboardPlugin);
+      //@ts-ignore
+      const clipboardData = JSON.stringify(cbPlugin.getgridData());
```

Comment:
nitpick: IMO it's prettier to write `cbPlugin["getgridData"]()` instead of a ts-ignore if we're dead set on totally ignoring typing, ts-ignore are ugly 😞